### DISCUSSION TITLE
Amazing job Jonas!

### DISCUSSION THREAD

**Discussion by OliHS** [jonas-hoebenreich/wc-donation-platform#51] [General]
Hi @jonas-hoebenreich , just a quick message to congrats you! your plugin is amazing! well done!

**Comment by jonas-hoebenreich** [comment]
thank you

**Comment by PoweredByConnect** [comment]
Just found this plugin, Thanks @jonas-hoebenreich  !  I was **_really_** not looking forward to getting locked in to GiveWP -- nightmare averted.  *Thank You So Much* for democratizing this important ecommerce feature.

**Comment by jonas-hoebenreich** [comment]
Thank you so much for your positive feedback. [If you like Donation Platform for WooCommerce and want to support the further growth and development of the donation plugin, please consider a 5-star rating on wordpress.org.](https://wordpress.org/support/plugin/wc-donation-platform/reviews/?filter=5#new-post) I just received a 1star rating and would really appreciate some positive reviews to balance that out :)
Best,
Jonas

**Reply by PoweredByConnect** [reply]
Kay, np!

**Comment by tasza** [comment]
Hi Jonas,
I am in the process of setting up your plugin for a non-profit organization. I just gave you a 5-star review - great work on the plugin and quite an easy set up for someone who has never used WordPress before ;)

Would you be so kind and approve my translations to Polish? I will add more tomorrow, if you give me a green light to do so. The thread on wordpress forum is unfortunately closed. Your plugin has all the features we need, but having a mix of English and Polish on a donation page is just suspicious and a no go for our campaign.

Thank you so much for your work!
Best,
Natasza

**Comment by jonas-hoebenreich** [comment]
Hi @tasza,

Thank you so much for your positive feedback and the 5-star review! I'm glad to hear that you found the plugin easy to set up, especially as a first-time WordPress user.

Regarding your request for translation approval, unfortunately, I am unable to approve translations directly. The approval process for translations is managed by the general translation editors for each language in the WordPress community. This ensures that all translations meet the quality and consistency standards set by WordPress.

However, you can use a plugin like Loco Translate to manage and apply translations directly on your website. This will allow you to have the Polish translations you need without waiting for the approval process: https://wcdp.jonh.eu/documentation/usage/internationalization-localization-translation/

If you have any other questions or need further assistance, feel free to reach out. Thank you again for your support and understanding.

Best,
Jonas

**Reply by tasza** [reply]
Thank you for the tips. I will try to use Loco Translate, I don't know how I've missed it. Have a nice week!
